tox.ini

```diff
@@ -6,6 +6,7 @@ envlist =
     py{27,35,36,37}-ansible{25,26,27,28}-{functional,unit}
     format-check
     doc
```

Comment:
@decentral1se can we integrate changelog generation with doc generation? In such a case, we'd not need `changelog-check` here.

Reply:
That's a fine idea for if the changelog generation was working smoothly but it is not even in place yet and there are a lot of steps to get it fully integrated. Meanwhile, documentation changes must remain easy to do. Let's do this later.